src/web.js

```diff
@@ -24,7 +24,25 @@ https.globalAgent.maxSockets = http.globalAgent.maxSockets = config.hostingEnvir
 
 const app = express();
 
-app.use(helmet());
+if(config.hostingEnvironment.hstsMaxAge){
+  app.use(helmet({
+    noCache: true,
+    frameguard: {
+      action: 'deny',
+    },
+    hsts: {
+      maxAge: config.hostingEnvironment.hstsMaxAge,
```

Comment:
use process.env.hstsMaxAge isstead

ask Lenin to create new appSetting hstsMaxAge

Reply:
By default helmet adds STS -> max-age=15552000; includeSubDomains. What do you want it set to in the appSettings?